### ISSUE TITLE
`NumberInput` allows non-numeric input

### ISSUE THREAD

**Issue by wei2912** [glints-dev/glints-aries#320]
`NumberInput` appears to allow non-numeric input (such as characters), although it should only allow numbers to be entered.

The following screenshot is from a modal I was coding:
![Screenshot from 2020-05-04 18-04-52](https://user-images.githubusercontent.com/2070423/80955371-c94c8b80-8e31-11ea-9af9-b1e0f8e4e903.png)

This behaviour can be replicated on http://aries.glints.design/?path=/story/input--numberinput as well.

**Comment by westwood846** [comment]
Did you just type them in? When I type nothing appears (except when typing `e` for some reason) xD

**Comment by westwood846** [comment]
Please state your OS and browser!

**Comment by wei2912** [comment]
> Did you just type them in?

Yep haha, I didn't copy paste the text. Though that works as well...

> Please state your OS and browser!

Oops, I forgot about that... Firefox 76.0, Ubuntu 18.04.
